Lenovo_Chromebook_300e.md

```diff
@@ -0,0 +1,129 @@
+## Xorg
+
+* The PowerVR GX6250 GPU is not supported at this time.  Therefore, no particular Xorg drivers are needed.  The built in modesetting driver will correctly detect the display.
+* xf86-input-libinput is preferred over evdev and synaptics for the best experience with the touchpad and touchscreen.
+
+## Audio
+Many, many channels must be unmuted and their volume turned up, so this handy script will make sure all are unmuted.
+```
+COUNT=0
+while [ $COUNT -lt 200 ]; do echo $COUNT; amixer -c 0 cset numid=$COUNT on; let COUNT=COUNT+1; done
+```
+
+Some of the channels must also have their volume increased.
+
+The following channels must be unmuted for the speakers to work.  The alsamixer utility (part of alsa-utils) can be used to do this by navigating with the arrow keys and pressing **m** to unmute the channel.
```

Comment:
Doing this manually is very dangerous. I once did this and ended up with bad smelling speakers.

I would use ucm files instead.